app/providers/AppContextProvider.tsx

```diff
@@ -41,6 +40,7 @@ export const AppProvider: React.FC<{
   const [isCompiling, setIsCompiling] = useState(false);
   const [currentFileFunctionCallResults, setCurrentFileFunctionCallResults] =
     useState<FunctionCallResult[] | undefined>(undefined);
+  const [chainId, setChainId] = useState<number>(base.id);
```

Comment:
TODO: we'll want to generalize this to picking your fork environment. Will require backend changes

Reply:
yeah was thinking about this. concluded that it was probably out of scope for this PR